src/test/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/integration/GoalControllerTest.kt

```diff
@@ -74,6 +74,7 @@ class GoalControllerTest : IntegrationTestBase() {
       title = "abc",
       areaOfNeed = areaOfNeedName,
       targetDate = LocalDate.now().toString(),
+      reminderDate = null,
```

Comment:
Why do we need this line?

Reply:
We don't need it, probably added it when I was playing around with the tests. Resolved in [ed561c7](https://github.com/ministryofjustice/hmpps-sentence-plan/pull/201/commits/ed561c788900d67665d85c4514de31fce8d64b6a)